tracker-neuralnet/CMakeLists.txt

```diff
@@ -1,23 +1,25 @@
 include(opentrack-opencv)
-find_package(OpenCV QUIET)
+find_package(OpenCV QUIET COMPONENTS imgproc core imgcodecs calib3d)
 find_package(OpenMP QUIET) # Used to control number of onnx threads.
-set(SDK_ONNX_LIBPATH "" CACHE FILEPATH "Full path of onnx library")
-
-if(OpenCV_FOUND AND SDK_ONNX_LIBPATH AND OpenMP_FOUND)
-    get_filename_component(ONNX_INCLUDE_DIR "${SDK_ONNX_LIBPATH}" DIRECTORY)
-    get_filename_component(ONNX_INCLUDE_DIR "${ONNX_INCLUDE_DIR}" ABSOLUTE)
-    set(ONNX_INCLUDE_DIR "${ONNX_INCLUDE_DIR}/../include")
+find_package(ONNXRuntime QUIET)
 
+if(OpenCV_FOUND AND ONNXRuntime_FOUND AND OpenMP_FOUND)
     otr_module(tracker-neuralnet)
-    target_include_directories(${self} SYSTEM PUBLIC 
-        ${OpenCV_INCLUDE_DIRS} "${ONNX_INCLUDE_DIR}")
+    target_include_directories(${self} 
+        SYSTEM PRIVATE 
+        ${OpenCV_INCLUDE_DIRS} 
+        ${ONNXRuntime_INCLUDE_DIRS}
+        )
     target_link_libraries(${self} 
-        opentrack-cv "${SDK_ONNX_LIBPATH}" opencv_imgproc opencv_core 
-        opencv_imgcodecs opencv_calib3d
-        OpenMP::OpenMP_C)
-
+        opentrack-cv 
+        ${ONNXRuntime_LIBRARIES} 
```

Comment:
Since you defined a target for the onnx library, just use that instead (`onnxruntime `)